modules/apps/site-initializer/site-initializer-liferay-learn/src/main/resources/site-initializer/style-books/learn/frontend-tokens-values.json

```diff
@@ -54,6 +54,14 @@
 		"cssVariableMapping": "color-neutral-10",
 		"value": "#282934"
 	},
+	"colorNeutral2": {
+		"cssVariableMapping": "color-neutral-2",
+		"value": "#E2E2E4"
```

Comment:
Could you double check these hex values?  Neutral 2 should be #394452 and Neutral 8 should be ##ebeef2

Reply:
In figma, it says that neutral 8 is #54555F
<img width="147" alt="image" src="https://user-images.githubusercontent.com/30430093/202592769-f9051ecc-15bf-4a07-bf13-d421c1b8ce3e.png">

neutral 2 is #E2E2E4
<img width="147" alt="image" src="https://user-images.githubusercontent.com/30430093/202592882-a8297b08-90a1-408e-ab8b-2127ef438243.png">